packages/dynamoose/lib/Table/index.ts

```diff
@@ -205,10 +207,10 @@ export class Table extends InternalPropertiesClass<TableInternalProperties> {
 					"arrayItemsMerger": utils.merge_objects.schemaAttributesMerger
 				})(...createTableAttributeParams);
 			},
-			"getHashKey": (): string => {
+			"getHashKey": () => {
```

Comment:
Any reason why we are removing these return types?

Reply:
Brought it back @https://github.com/dynamoose/dynamoose/pull/1538/commits/2e5cb52d97f69269dc961e9b9bf2b7a04c804a8d